runtime/common/src/crowdloan.rs

```diff
@@ -513,17 +519,21 @@ decl_module! {
 						sub: index,
 					});
 
-					// Care needs to be taken by the crowdfund creator that this function will succeed given
-					// the crowdfunding configuration. We do some checks ahead of time in crowdfund `create`.
-					let _ = <slots::Module<T>>::handle_bid(
+					// Care needs to be taken by the crowdloan creator that this function will succeed given
+					// the crowdloaning configuration. We do some checks ahead of time in crowdloan `create`.
+					let result = <slots::Module<T>>::handle_bid(
 						bidder,
 						auction_index,
 						fund.first_slot,
 						fund.last_slot,
 						fund.raised,
 					);
+
+					Self::deposit_event(RawEvent::HandleBidResult(index, result));
 				}
 			}
+
+			0
```

Comment:
You added a benchmark for this. Why not using the results here?

Reply:
I didnt put any benchmark integration yet because I have not run them. I still have updates to slots which this pallet is tightly coupled to, so benchmarks wont matter until that is also done.